client/MentorPanel/src/components/video-panel.js

```diff
@@ -1,15 +1,17 @@
 import React from "react"
 import { useSelector, useDispatch } from 'react-redux';
+import { CircularProgress } from '@material-ui/core';
 
-import { setCurrentMentor } from '../redux/actions'
-import { videoUrl, RESPONSE_CUTOFF } from '../api/api'
+import { selectMentor } from '../redux/actions'
+import { RESPONSE_CUTOFF } from '../api/api'
 
 import VideoThumbnail from "./video-thumbnail"
 
 const VideoPanel = () => {
   const dispatch = useDispatch()
-  const mentor = useSelector(state => state.cur_mentor)
-  const mentors = useSelector(state => state.mentors)
+  const mentor = useSelector(state => state.current_mentor)
+  const mentors = useSelector(state => state.mentors_by_id)
+  const question = useSelector(state => state.current_question)
 
   const isDisabled = (id) => {
     return mentors[id].confidence <= RESPONSE_CUTOFF
```

Comment:
move the knowledge for this to the action, so that here you can just check a calculated property on the state, e.g. `mentors[id].has_valid_answer`